src/order/filter/pointwise.lean

```diff
@@ -84,19 +84,18 @@ end one
 section mul
 variables [has_mul α] [has_mul β] {f f₁ f₂ g g₁ g₂ h : filter α} {s t : set α}
 
-@[to_additive] instance : has_mul (filter α) := ⟨map₂ (*)⟩
+/-- The filter `f * g` is generated by `{s * t | s ∈ f, t ∈ g}` in locale `pointwise`. -/
+@[to_additive "The filter `f + g` is generated by `{s + t | s ∈ f, t ∈ g}` in locale `pointwise`."]
+protected def has_mul : has_mul (filter α) :=
+⟨λ f g, { sets := {s | ∃ t₁ t₂, t₁ ∈ f ∧ t₂ ∈ g ∧ t₁ * t₂ ⊆ s}, ..map₂ (*) f g }⟩
```

Comment:
Why is this better than the old speliing?

Reply:
cf the PR description. `obtain ⟨t₁, t₂, h₁, h₂, h⟩ : s ∈ f * g` now gives `h : s ⊆ t₁ * t₂` instead of `h : s ⊆ set.image2 (*) t₁ t₂`.

Reply:
Irrelevant now